app/jobs/scheduled/resend_validation_emails.rb

```diff
@@ -0,0 +1,22 @@
+module Jobs
+  class ResendValidationEmails < Jobs::Scheduled
+    every 2.days
```

Comment:
This should run more frequently otherwise we might send reminder after (almost) 4 days.

```suggestion
    every 2.hours
```